client/my-sites/menus/controller.js

```diff
@@ -20,64 +20,58 @@ import { renderWithReduxStore } from 'lib/react-helpers';
 
 const sites = sitesFactory();
 
-var controller = {
+export function menus( context ) {
```

Comment:
Yeah, it could. I was aiming for consistency with the more generic controllers (e.g. the ones from `my-sites/controller`) which aren't `default` either (because there are more than one), but I guess it's fair to make this one `default`.